src/Compilation.zig

```diff
@@ -1288,6 +1302,7 @@ pub fn create(gpa: *Allocator, options: InitOptions) !*Compilation {
             .use_lld = use_lld,
             .use_llvm = use_llvm,
             .system_linker_hack = darwin_options.system_linker_hack,
+            .libc_stub_path = libc_stub_path,
```

Comment:
Why does there need to be any code here in Compilation.zig? There is no user input configurability here, so the link code could just do this logic. No need to add a field to link.Options.

Reply:
Excellent point. I think I even had the logic in the linker proper before, but then couldn't decide which approach would be best and ended up with this. Anyhow, now the logic is moved back into the linker!